frontend/.env.production

```diff
@@ -1,3 +1,3 @@
-VUE_APP_FRONTEND=https://www.theurlist.com
-VUE_APP_FUNCTION_KEY=SteUyT0o2YSt4JW8k2qZ5ShTBcQYQj1pxE8cO7oTrYI4ps92/53o9w==
-VUE_APP_BACKEND=https://www.theurlist.com
+VUE_APP_FRONTEND=https://www.theurlist.com

```

Comment:
we should remove this file and set the value on deploy (probably relate to the task Jafar is working on

Reply:
Will keep this out for not to reduce further complicating this PR.

Reply:
yes, this is the task that I am taking care of